test/styles/RRR.R

```diff
@@ -3,19 +3,23 @@
 ### Function declarations
```

Comment:
This section is not only about function declarations, but also about function calls.